internal/governor/governor.go

```diff
@@ -91,11 +92,13 @@ func update(w io.Writer, ud updateData) error {
 
 type WaitError struct {
 	Duration time.Duration
+	Reason   string
 }
 
-func newWaitError(duration time.Duration) error {
+func newWaitError(duration time.Duration, reason string) error {
 	return WaitError{
 		Duration: duration,
+		Reason:   reason,
```

Comment:
Since we now have this extra info, would it make sense to add to the WaitError `Error()` method's string?